### ISSUE TITLE
Dark mode toggle not properly working on pages with lordicons

### ISSUE THREAD

**Issue by MxNl** [jmgirard/lordicon#2]
Hi,

first of all thanks for this really nice extension! I am currently creating a quarto book. I noticed that when including a lordicon with this extension in a chapter with 
```
{{< li lupuorrc trigger=loop delay=5000 speed=1.0 colors=secondary:#619ab3 >}}
```
the dark mode toggle only affects the navbar. In chapters where there is no lordicon, the dark mode just works fine.
I hope you can easily reproduce this.

This is how my _quarto.yml file looks like:
```
project:
  type: book
  output-dir: docs

book:
  title: "Applied Programming for Geosciences"
  subtitle: "From Data Cleaning to Publishing"
  author: "Name Papame"
  date: "3.2.2023"
  chapters:
    - index.qmd
    - intro.qmd
    - rbasics.qmd
  repo-actions: [edit]
  downloads: [pdf, epub]
format:
  html:
    theme:
      light: cosmo
      dark: darkly
  pdf:
    documentclass: scrreprt
  epub:
    cover-image: images/logo.svg

comments:
  hypothesis: true

css: style.css

bibliography: references.bib

editor: visual
```

I would appreciate if you could look into a fix for that. Thanks a lot!

**Comment by jmgirard** [comment]
If you can make a minimal working example (MWE) and share the `_quarto.yml` and `.qmd` files, I will take a look. Sounds like it might be a Quarto issue more than a lordicons issue but the MWE will help regardless.

**Comment by MxNl** [comment]
Hello,
thanks for looking into it. 
Here the MWE:

_quarto.yml:
```
project:
  type: book

book:
  title: "reprex_quartobook"
  author: "Jane Doe"
  date: "8.2.2023"
  chapters:
    - index.qmd
    - intro.qmd

format:
  html:
    theme:
      light: cosmo
      dark: darkly
```

index.qmd:
```
# Preface {.unnumbered}

This is a Quarto book.

To learn more about Quarto books visit <https://quarto.org/docs/books>.

{{< li lupuorrc trigger=loop delay=5000 speed=1.0 colors=secondary:#619ab3 >}}
```

**Comment by jmgirard** [comment]
Note that I was able to reproduce the problem and opened a discussion under quarto-cli:
https://github.com/quarto-dev/quarto-cli/discussions/4341

**Comment by jmgirard** [comment]
This should be fixed now in [v0.3.0](https://github.com/jmgirard/lordicon/releases/tag/v0.3.0) by https://github.com/jmgirard/lordicon/commit/4dd3edc5668d245730f71cc13a6d6ba83ee19fe6. Thanks for calling it to my attention!

**Comment by MxNl** [comment]
Perfect! Thanks a lot!
